akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -750,6 +750,9 @@ object Source {
    * @param bufferSize size of buffer in element count
    * @param overflowStrategy Strategy that is used when incoming elements cannot fit inside the buffer
    */
+  @Deprecated
+  @deprecated("Use queue without `overflowStrategy` parameter instead.", "2.6.14")
```

Comment:
@alex-ninja Great work in this PR. As you have seen in the [original ticket](https://github.com/akka/akka/issues/29801#issuecomment-799408986) we would like to take a step back and don't rush this (and we don't have bandwidth for it right now). 

Let's revert the deprecated annotations also, so we don't deprecate something and don't have good answers of what to use instead. The test changes are still great, thanks.

Reply:
@patriknw That discussion makes perfect sense to me. Maybe we'll do more changes next time. Meanwhile, I've rolled back the changes related to the `Source.queue` deprecation.